Python/peephole.c

```diff
@@ -250,12 +250,19 @@ PyCode_Optimize(PyObject *code, PyObject* consts, PyObject *names,
     lnotab = (unsigned char*)PyBytes_AS_STRING(lnotab_obj);
     tabsiz = PyBytes_GET_SIZE(lnotab_obj);
     assert(tabsiz == 0 || Py_REFCNT(lnotab_obj) == 1);
-    if (memchr(lnotab, 255, tabsiz) != NULL) {
-        /* 255 value are used for multibyte bytecode instructions */
-        goto exitUnchanged;
+
+    /* 255 values in the instruction pointer are used for multibyte
```

Comment:
Do we really need this limitation? What if remove this check at all?

Reply:
Then multibyte bytecode won't be handled correctly (although running the test suite did not reveal any failure). I don't know if this has different side effects that we are not considering, the function declares that it does not handle those cases:

```
   To keep the optimizer simple, it bails when the lineno table has complex
   encoding for gaps >= 255.
```

Personally, I don't feel comfortable to drop the requirement entirely, but if you think is ok we can do it.

Reply:
If you want to support multibyte instruction pointer delta, the code should be fix. Removing the check is not enough. Otherwise, the check would be pointless :-)